content/doc/developer/publishing/documentation.adoc

```diff
@@ -58,6 +58,7 @@ Documentation examples:
 
 link:https://wiki.jenkins.io[Jenkins Wiki] can still be used for previously created plugins or for new plugins if there are reasons to do so.
```

Comment:
Considering the actual context, I think this sentence is confusing

Reply:
Feel free to submit a follow-up PR, this text was created before the announcement by @rtyler and I still do not see consensus there. Also, @daniel-beck was against marking Wiki as deprecated when this text was suggested last week